src/Listeners/GenerateModelHelper.php

```diff
@@ -45,11 +46,10 @@ public function handle(CommandFinished $event)
 
         self::$shouldRun = false;
 
-        $parameters = $this->config->get('ide-helper.post_migrate');
-        $this->artisan->call(
-            is_array($parameters) ? 'ide-helper:models' : 'ide-helper:models ' . $parameters,
-            is_array($parameters) ? $parameters : [],
-            $event->output,
-        );
+        $commands = $this->config->get('ide-helper.post_migrate');
+
+        foreach (Arr::wrap($commands) as $command) {
```

Comment:
The `Arr::wrap` I think will break when using the current `false`, can you check?

`Arr::wrap` has a special case for `null` coercing to an empty array (which would be fine here), but otherwise it wants to process the defaults as `[false]` when I read this correctly.

Maybe it would be just easier to make `post_migrate` always an array, also in the config, with an empty one but one commented example which could user help to quicker start using it:
```php
'post_migrate' => [
    // 'ide-helper:models --nowrite',
],
```

Reply:
Further, please also add the default to `$this->config->get()`, not everyone upgrading the library immediately upgrades the config files (though incidentally, in _this_ case the `null` default would just work but it just would seem inconsistent then).

Reply:
It should not matter since the [event listener is not registered when falsy values are provided](https://github.com/barryvdh/laravel-ide-helper/pull/1163/files#diff-08724b347a720a68357b78ce96d5c46a5fcb0c90bc3784594620a0af65e2bf45R38)

Reply:
Simplifying the config makes sense so I did that. Also the package config is always merged with the user config so there shouldn't be an event where the config entry is missing.

Reply:
> Also the package config is always merged with the user config

That's interesting, we definitely had issues in the past due to missing config values and no defaults in the code 🤷‍♀️

Reply:
Interesting, this [line should](https://github.com/barryvdh/laravel-ide-helper/blob/master/src/IdeHelperServiceProvider.php#L57) prevent that.